css/style.css

```diff
@@ -16,4 +15,107 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ .header,
+ .ma-img,
+ .footer,
+ .icon{
+    padding: 0 1.5rem;
+    max-width: auto;
+    margin: 0 auto;
+ }
+ .header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+ }
+ nav{
+    display: flex;
+    width: 100%;
+ }
+.navbar{
+    display: flex;
+    list-style: none;
+    font-size: normal;
+    color: lightgrey;
+    padding-left: 8rem;
```

Comment:
The header is more to the left than it should be, should be right aligned as shown in the picture of the sample site
![image](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/assets/21955614/bdf2662f-f9a2-44b3-bc99-1c301773e77f)